_sass/blocks/_footer.scss

```diff
@@ -24,19 +24,24 @@ footer,
   padding-bottom: ($base-padding-extra * 1.5);
   padding-top: ($base-padding-extra * 1.5);
 
-  .fa-circle {
-    color: $neutral-gray;
-  }
-
-  .fa-inverse {
-    color: $light-black;
-  }
+  // .fa-circle {
```

Comment:
Is there a reason this stuff is commented out rather then just removed?

Reply:
To have handy if we re-instate social. Happy to kill.